### PR TITLE
add mondaycoderc check

### PR DESCRIPTION
**Pull request by danielva-monday**

### CODE REVIEW

File: src/services/schemas/mondaycoderc-schema.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import { z } from 'zod';
+
+export const mondaycodercSchema = z
```
Comment (gregra81) [inline src/services/schemas/mondaycoderc-schema.ts:3]: please add a few unit tests for this

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mondaycom/apps-cli",
-  "version": "4.1.1",
+  "version": "4.1.2",
```
[the remark this answers is not in the document]
Comment (danielva-monday) [reply package.json]: done

File: src/services/schemas/mondaycoderc-schema.ts (hunk 1 above)
Comment (danielva-monday) [reply src/services/schemas/mondaycoderc-schema.ts:3]: added

File: src/services/__tests__/mondaycoderc-schema.test.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import { mondaycodercSchema } from 'services/schemas/mondaycoderc-schema';
+
+describe('mondaycodercSchema Validation', () => {
```
Comment (gregra81) [inline src/services/__tests__/mondaycoderc-schema.test.ts:3]: Nice. Also let's add one that throws if someone provides an unsupported engine

Comment (danielva-monday) [reply src/services/__tests__/mondaycoderc-schema.test.ts:3]: added a test and a new error message for invalid runtime

**Review by gregra81 — APPROVED** [review]
Amazing work 👏 💯

**Pull request merged**